### PR TITLE
add support for send directrly an telegram message

### PR DESCRIPTION
**Pull request by steffjenl**
Hello,

I read my energy meter with https://github.com/dennissiemensma/dsmr-reader and want to report also to my Homey.

I have created an separated webhook that can read raw telegram messages. And an plugin for dsmr-reader to push data to homey.

https://gist.github.com/steffjenl/31bd083eeb9d0be04375b7695b9f2eaf

### CODE REVIEW

File: lib/dsmrreader.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+                                'unit': 'kW',
+                            },
+                        },
+                    },
+                },
+            }
+        }
+
+        // backwards compatibility
+        data.gas = {
```
Comment (koktaildotcom) [inline lib/dsmrreader.js:107]: This option is deprecated and not needed anymore

File: .gitignore [hunk 2]
```diff
@@ -18,3 +18,5 @@
 
 .idea/libraries
 .idea/
+node_modules/
+package-lock.json
```
Comment (koktaildotcom) [inline .gitignore:22]: The `package-lock.json` shoultn't be in the gitignore file.

File: lib/dsmrreader.js (hunk 1 above)
Comment (steffjenl) [reply lib/dsmrreader.js]: I have removed:
`        // backwards compatibility
        data.gas = {
            'reading': null,
        }`

File: .gitignore (hunk 2 above)
Comment (steffjenl) [reply .gitignore]: removed package-lock.json from gitignore and added file package-lock.json to repo

**Comment by steffjenl** [comment]
Yes, i did make the changes now. Please review the changes! Thank you!

**Comment by steffjenl** [comment]
I added some more information about the endpoint and installation in the README files.

**Comment by koktaildotcom** [comment]
Your PR will be in release v3.1.3

**Pull request merged**